### PR TITLE
Upgrade to kops 1.18.3, terraform 0.12.29 and kubectl version 1.18.15

### PR DESCRIPTION
**Pull request by angeloskyratzakos**
#### Summary
Updating kops version to 1.18.3, terraform 0.12.29 and kubectl version 1.18.15
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28911

#### Release Note

```release-note
updates to use terraform 0.12.29, kops 1.18.3 and kubectl version 1.18.15
```

### CODE REVIEW

**Review by stylianosrigas — APPROVED** [review]
Yesss finally, LGTM 👌

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The following is required to properly run the cloud server.
 1. Install [Go](https://golang.org/doc/install)
 2. Install [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html) version v0.11.14
```
Comment (gabrieljackson) [inline README.md:30]: Looks like we need to update the version recommendation here as well.

**Review by gabrieljackson — COMMENTED** [review]
Looks good, but there are two binary files checked in with this commit that probably shouldn't be. Let's remove them and then we should be good to go!

File: README.md (hunk 1 above)
Comment (angeloskyratzakos) [reply README.md:30]: You are absolutely right! thanks for the suggestion!

**Comment by angeloskyratzakos** [comment]
> Looks good, but there are two binary files checked in with this commit that probably shouldn't be. Let's remove them and then we should be good to go!

ahhh my bad I was checking if the makefile works fine with new versions. I will update the gitignore

**Review by gabrieljackson — APPROVED** [review]
👍

**Pull request merged**